src/com/csciOOD/Sprite.java

```diff
@@ -1,14 +1,19 @@
 package com.csciOOD;
 
-public class Sprite {
+import javax.swing.*;
+import java.awt.*;
+
+public class Sprite extends JPanel {
```

Comment:
I like this and I think we should keep it this way, but I also just want to bookmark the idea of offloading the JPanel work to the GameObject **IF** it makes things like collision detection a little easier.